config/settings.py

```diff
@@ -69,6 +69,7 @@
     "psqlextra",
     "rest_framework",
     "rest_framework_simplejwt",
+    "djangoql",
```

Comment:
I see it was already broken before you by the `pg*` stuff but could we sort this alphabetically so humans can more easily navigate through it when looking for something? :smiley: